Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -495,10 +495,13 @@ private void RestoreList(string Name, IObservableList v, bool isLocalBackup = fa
             }
         }
 
-        public void RestoreFromBackup(bool isLocalBackup = false)
+        public void RestoreFromBackup(bool isLocalBackup = false, bool clearBackup = true)
         {
             RestoreBackup(isLocalBackup);
-            ClearBackup(isLocalBackup);
+            if (clearBackup)
```

Comment:
why this flag was added?

Reply:
just added this flag if at some point user does not want to the clear backup then this flag can be used